src/features/cli/ReverseShellConnection.jsx

```diff
@@ -0,0 +1,112 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import styled from "@emotion/styled";
+import { ExpandMore } from "@mui/icons-material";
+import {
+  Card, CardContent, Collapse, CardHeader, IconButton, TextField, Button,
+} from "@mui/material";
+import { Box } from "@mui/system";
+import { useCallback } from "react";
+
+const ExpandMoreWrapper = styled((props) => {
+  const {
+    expand, ...other 
+  } = props;
+  // eslint-disable-next-line react/jsx-props-no-spreading
+  return <IconButton {...other} />; // TODO: Review: How to fix?
+})(({
+  theme, expand, 
+}) => ({
+  transform: !expand ? "rotate(0deg)" : "rotate(180deg)",
+  marginLeft: "auto",
+  transition: theme.transitions.create("transform", { duration: theme.transitions.duration.shortest }),
+}));
+
+export default function ReverseShellConnection({
+  connected,
+  onClick, 
+}) {
+
+  const hostInputRef = React.useRef();
+  const portInputRef = React.useRef();
+
+  const [expanded, setExpanded] = React.useState(false);
+
+  const handleExpand = useCallback(() => setExpanded(!expanded), [expanded]);
+  const handleClick = useCallback(() => onClick({
+    host: hostInputRef.current.value,
+    port: portInputRef.current.value,
+  }), [onClick, hostInputRef, portInputRef]);
+
+  return (
+    <Card>
+      <CardHeader
```

Comment:
Generally I think it would be nice if the whole header could act as an expander, so that the user can click anywhere instead of just the icon.